test/config_tests.ml

```diff
@@ -47,7 +109,40 @@ testpass
 remote 10.0.0.1|} in
   Alcotest.(check (result conf_map pmsg)) "basic conf works"
     (Ok minimal_config)
-    (parse_noextern basic)
+    (parse_noextern_client basic)
+
+let minimal_server_config =
+  let open Openvpn.Config in
+  empty
+  |> add Dev (`Tun ,(Some "tunnel"))
+  |> add Ping_interval `Not_configured
+  |> add Cipher "AES-256-CBC"
+  |> add Ping_timeout (`Restart 120)
+  |> add Renegotiate_seconds 3600
+  |> add Bind (Some (Some 1195, None))
```

Comment:
See my review on #38: https://github.com/roburio/openvpn/pull/38#discussion_r399712089
#38 has fixes for most of this, e.g. https://github.com/roburio/openvpn/pull/38/commits/64ebf6f6c73ac828fc91191fe996ded5f5b313e0
and

Reply:
your comment in #38 is about `Config.Defaults.server` (this is Config_test.minimal_server_config -- and actually unused since we don't have a parse_server_config yet) -- I'd be happy to get rid of the minimal_*_config in test and reuse Config.Defaults.yy in a subsequent PR. in the meantime, I just deleted the commented out and unused code in 17a995c5a15d83c6952701138e1ea3104a4d8539.